# MyDict.msg

Header header
string causal_model